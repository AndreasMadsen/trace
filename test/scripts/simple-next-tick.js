'use strict';

//
// test process method (nextTick)
//

process.nextTick(function callback() {
  throw new Error('trace');
});
