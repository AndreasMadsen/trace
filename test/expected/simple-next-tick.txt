/scripts/simple-next-tick.js:8
  throw new Error('trace');
  ^

Error: trace
    at callback (/scripts/simple-next-tick.js:8:9)
    at ____________________:0:0
    at Object.<anonymous> (/scripts/simple-next-tick.js:7:9)
